---
float64[] box_centers_y
float64[] pen_centers_y